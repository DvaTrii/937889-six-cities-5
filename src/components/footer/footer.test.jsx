import React from "react";
import {BrowserRouter} from "react-router-dom";
import renderer from "react-test-renderer";
import Footer from "./footer";

it(`Should Footer render properly`, () => {
  const tree = renderer
    .create(
        <BrowserRouter>
          <Footer/>
        </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
