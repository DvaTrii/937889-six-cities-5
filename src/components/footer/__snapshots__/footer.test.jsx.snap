// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Footer render properly 1`] = `
<footer
  className="footer container"
>
  <a
    className="footer__logo-link"
    href="/"
    onClick={[Function]}
  >
    <img
      alt="6 cities logo"
      className="footer__logo"
      height="33"
      src="img/logo.svg"
      width="64"
    />
  </a>
</footer>
`;
